### DISCUSSION TITLE
[Bug] UQL replace_string function is not working as expected

### DISCUSSION THREAD

**Discussion by fwdolegp** [grafana/grafana-infinity-datasource#1109] [General]
## **Describe the bug**

Function  replace_string is not working as expended

## **Steps to reproduce**

Use UQL

parse-json
| project "app"=replace_string("app","v2","latest")

**Note:** bugs will be fixed only if enough information provided to reproduce the issue

## **Sample Data**

```javascript
[
{ 
    "app": "1.0.0-v2"
}
]
``` 

## **Version Details:**

- Grafana version : grafana/grafana:11.3.0
- Plugin version : 2.11.1

**Comment by yesoreyeram** [comment]
You need to use single quote for referring string.

```kusto
parse-json
| project "app"=replace_string("app",'v2','latest')
```

![Image](https://github.com/user-attachments/assets/0e3fb73c-8f37-41a0-9432-261e79359fea)

**Comment by Harsh130704** [comment]
@yesoreyeram : I am using below query for iif function but its not working , could you please give us the correct fornat for this conditional query

```kusto
parse-json
| jsonata "components[id in $Login_format ]"
| project "status" = iif("status" == 'operational',1,0)
````

**Comment by yesoreyeram** [comment]
> [@yesoreyeram](https://github.com/yesoreyeram) : I am using below query for iif function but its not working , could you please give us the correct fornat for this conditional query parse-json | jsonata "components[id in $Login_format ]" | project "status" = iif("status" == 'operational',1,0)

Hiya. I recommend you to search for existing answers in [Grafana community forum](https://community.grafana.com/search?q=infinity%20order%3Alatest) and in [github discussions](https://github.com/grafana/grafana-infinity-datasource/discussions). If you don't see an existing answer, create a new post there. Github issues are not correct place of asking help.
